releases/v0.4.0.md

```diff
@@ -0,0 +1,23 @@
+# Release Notes for v0.4.0 (release candidate)
+
+## Features
+
+- Support [Evaluation (beta)](../evaluation.mdx) tool to evaluate the performance and reliability of the Chat Engine’s outputs.
+  - Current support key metrics:
+    - Factual Correctness
+    - Semantic Similarity
+- Support new LLM providers
+  - Gitee AI
```

Comment:
```suggestion
  - Gitee AI
  - OpenRouter
```

Reply:
The rest are not newly added in this release.